modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -777,7 +848,7 @@ void PathBoundsDecider::TrimPathBounds(
 }
 
 void PathBoundsDecider::PathBoundsDebugString(
-      const std::vector<std::tuple<double, double, double>>* path_boundaries) {
+    const PathBoundary* path_boundaries) {
```

Comment:
As my previous comment, use const reference here.

Reply:
fixed this.